curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-hobbies-and-interests/657da34294cbf695b2bfe889.md

```diff
@@ -3,17 +3,14 @@ id: 657da34294cbf695b2bfe889
 title: Task 26
 challengeType: 19
 dashedName: task-26
-audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
 ---
 
-<!--
-AUDIO REFERENCE: 
-Linda: I do! I've liked riding my bike since my childhood. It's not just a hobby; it's a way of life to me.
--->
+<!-- (Audio) Linda: I do! I've liked riding my bike since my childhood. It's not just a hobby; it's a way of life to me. -->
 
 # --description--
 
-The Present Perfect Tense can be used with the word `since`. People use `since` to talk about when something started and is still happening now. For example, `I have liked painting since I was a child.` This means they started liking painting when they were a child and they still like it now.
+The Present Perfect Tense can be used with the word `since`. People use `since` to talk about *when* something that is still happening now started. For example, `I have liked painting since I was a child.` This means they started liking painting when they were a child and they still like it now. Remember: you can abbreviate the auxiliary verb `have` as `'ve`.
```

Comment:
```suggestion
The Present Perfect Tense can be used with the word `since` to discuss the starting point of ongoing actions or conditions. For instance, `I have liked painting since I was a child.` implies that the speaker began to enjoy painting in their childhood and continues to do so. Note: the auxiliary verb `have` can be contracted to `'ve`.
```